### ISSUE TITLE
after years, NO ONE can fix the bug "AccessibilityInfo" ???

### ISSUE THREAD

**Issue by roytan883** [facebook/react-native#23906]
**after years**, lots of commits try to fix it, but the **`AccessibilityInfo`** bug still exists in every major release: 0.56, 0.57, 0.58, ..... **funny** ... 

**simplest To Reproduce:**
```
$ react-native init bugProject --version 0.58.6
$ cd bugProject
$ react-native dependencies --entry-file index.js
```
**result:**
```
Unable to resolve module `AccessibilityInfo` 
from `.../node_modules/react-native/Libraries/react-native/react-native-implementation.js`: 
Module `AccessibilityInfo` does not exist in the Haste module map
```

**Comment by karanjthakkar** [comment]
@roytan883 `AccessbilityInfo` is a module that has different implementation files for ios and android. So for your command to work, you need to provide the platform for which you want to generate dependencies.

If you change your command to `react-native dependencies --entry-file index.js --platform ios`, this will work as expected. Similary if you provide platform as `android`, it will give you deps for android.

**Comment by karanjthakkar** [comment]
@kelset This can be closed.

**Comment by roytan883** [comment]
@kageurufu Thanks for you reply. 
The `react-native dependencies --entry-file index.js --platform android` works now. 
But `yarn run metro build index.js --out index.bundle --platform android` still got this error !

```
Loading dependency graph...
 BUNDLE  [android, prod] ./index.js ░░░░░░░░░░░░░░░░ 0.0% (0/1)Error: Unable to resolve module `AccessibilityInfo` from `D:\demo\no
de_modules\react-native\Libraries\react-native\react-native-implementation.js`: Module `AccessibilityInfo` does not exist in the Haste module map
```

RN version is : 0.58.6
metro version is : 0.53.0

**Comment by karanjthakkar** [comment]
@roytan883 I don't know about this. Why would you use metro directly instead of using `react-native bundle` which calls build under the hood? Try this instead: `react-native bundle --entry-file index.js --platform android --bundle-output index.out.js`

**Comment by roytan883** [comment]
@karanjthakkar Because i want use `metro-visualizer` to analyse my bundle. But the `react-native run-android` internal use metro `0.49` which not support by `metro-visualizer`. So i can't use `react-native` CLI. 

Then I just manually installed `metro 0.53.0`. As `metro ` README says it can build and serve RN directly, 
i run this command `yarn run metro build index.js --out index.bundle --platform android`, or `yarn run metro serve`, both at the end show this error:
```
Loading dependency graph...
 BUNDLE  [android, prod] ./index.js ░░░░░░░░░░░░░░░░ 0.0% (0/1)Error: Unable to resolve module `AccessibilityInfo` from `D:\demo\no
de_modules\react-native\Libraries\react-native\react-native-implementation.js`: Module `AccessibilityInfo` does not exist in the Haste module map
```

**Comment by karanjthakkar** [comment]
@roytan883 Seems like a metro issue. Would be a good idea to raise this in the metro repository. This is not the right place for this specific error and you probably won't find any resolution here.

**Comment by roytan883** [comment]
@karanjthakkar seems the `metro` is inactive about issues. Anyway, thanks for you help , i close this right now.
